swagger/src/main/scala/org/http4s/rho/swagger/models.scala

```diff
@@ -187,6 +187,15 @@ object models {
     , vendorExtensions : Map[String, Any]  = Map.empty
     ) {
 
+    def operations: List[Operation] =
```

Comment:
Not mandatory for merging, and I know this can be a contentious issue, but my personal preference is to return a `Seq` unless its obvious that a `List` poses some unique advantages.